core/modules/views_ui/views_ui.admin.inc

```diff
@@ -1254,6 +1258,10 @@ function views_ui_render_display_top($view, $display_id) {
         'title' => t('clone view'),
         'href' => "admin/structure/views/view/$view->name/clone",
       ),
+      'export' => array(
+        'title' => t('export view'),
+        'href' => 'admin/config/development/configuration/single/export/Views/views.view.' . $view->name,
+      ),
```

Comment:
Could you add a `module_exists('config')` and permissions check (`synchronize configuration`) here?